### DISCUSSION TITLE
请问 MIPS64 (Octeon 5020) 下载哪个包？

### DISCUSSION THREAD

**Discussion by qifei** [shadowsocks/shadowsocks-rust#679] [General]
可以用这个吗？ shadowsocks-v1.12.0.mips-unknown-linux-musl.tar.xz

**Comment by zonyitoo** [comment]
You may have to compile it yourself.

**Reply by zonyitoo** [reply]
You can try all those pre-built mips* releases and see if it can run on your CPU.

**Comment by qifei** [comment]
谢谢。
我看到有好多架构的包，以为 MIPS 包括了 MIPS64
